docs/en-US/component/button.md

```diff
@@ -123,45 +123,59 @@ button/custom
-| bg<VersionTag version="2.2.0" />   | determine whether the text button background color is always on         | boolean               | —                                                             | false   |
-| link<VersionTag version="2.2.1" /> | determine whether it's a link button                                    | boolean               | —                                                             | false   |
-| round                              | determine whether it's a round button                                   | boolean               | —                                                             | false   |
-| circle                             | determine whether it's a circle button                                  | boolean               | —                                                             | false   |
-| loading                            | determine whether it's loading                                          | boolean               | —                                                             | false   |
-| loading-icon                       | customize loading icon component                                        | `string \| Component` | —                                                             | Loading |
-| disabled                           | disable the button                                                      | boolean               | —                                                             | false   |
-| icon                               | icon component                                                          | `string \| Component` | —                                                             | —       |
-| autofocus                          | same as native button's `autofocus`                                     | boolean               | —                                                             | false   |
-| native-type                        | same as native button's `type`                                          | string                | button / submit / reset                                       | button  |
-| auto-insert-space                  | automatically insert a space between two chinese characters             | boolean               |                                                               | —       |
-| color                              | custom button color, automatically calculate `hover` and `active` color | string                |                                                               | —       |
-| dark                               | dark mode, which automatically converts `color` to dark mode colors     | boolean               |                                                               | false   |
-
-## Button Slots
+## Button API
+
+### Attributes
```

Comment:
When a page contains multiple components, it is not recommended to omit the component name of the prefix, which will generate a wrong code hint file.

```diff
- ### Attributes
+ ### Button Attributes
```

Reply:
You are right, I will add this part, and make it a document specification later.